### PR TITLE
Added graphing of time series data when location is clicked 

### PR DESCRIPTION
**Pull request by aed3**

### CODE REVIEW

**Comment by camjc** [comment]
Excited to get something like this off the ground. @lazd was looking at something like this too. Whats the plan and how can I help out?

**Comment by camjc** [comment]
Probably need to revert the cache? Seems like this pr doesn’t mean to touch it?

**Comment by aed3** [comment]
> Probably need to revert the cache? Seems like this pr doesn’t mean to touch it?

You're right, but I'm not sure how to do that.

**Comment by aed3** [comment]
But the feature works 100% now, on mobile and everything. I can save the work for COVID Atlas, but don't think it would hurt to add to the current website @lazd

**Comment by lazd** [comment]
Yeah this looks good, we can merge!

File: site/index.html [hunk 1]
```diff
@@ -194,11 +196,17 @@ <h2 class="spectrum-Heading spectrum-Heading--M" id="spec">What does a good sour
...
+            <div id="graph-container">
+              <canvas id="graph"></canvas>
             </div>
-            <div id="map"></div>
+            <span onclick="document.getElementById('graph-elements').style.visibility='hidden'" id="graph-close">×</span>
+          </div>
+          <div class="cds-Map-controls">
           </div>
+          <div id="map"></div>
         </div>
```
Comment (lazd) [inline site/index.html:210]: Unclosed `<div>` broke viewer #820 :'(

**Pull request merged**